python/ray/tune/logger/logger.py

```diff
@@ -201,6 +201,7 @@ def log_trial_end(self, trial: "Trial", failed: bool = False):
                 trial_loggers[trial].close()
 
 
+@PublicAPI(stability="alpha")
```

Comment:
Let's make this DeveloperAPI, regular users don't need this

```suggestion
@DeveloperAPI
```